java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -49,16 +44,14 @@ public void startSession_defaultConfig_usWestDataCenter() {
 
     @Test
     public void getUserName_usernameSetInEnvironmentVariable_returnsValue() {
```

Comment:
Hmmm, something doesn't seem right here. If the user name is set in Env variables, why would it `assertNotEquals` to the expected value which is "test-name"? Shouldn't it be the opposite?

Reply:
yeah, this is a dumb test. I delete it when merging in #94 when we can evaluate that ENV are getting used by evaluating the public `getSauceURL` where it actually matters.

Reply:
actually, this is redundant with the tests
`usesENVAsDefaultUsername` and `usesENVAsDefaultAccessKey`

Reply:
of course with the change I made above, that makes those 2 tests not work until we have #94 implemented. Merge that one in and I'll rebase this so we can un-ignore these tests. :)